content/blog/2022-04-26-april-22-community-gems.md

```diff
@@ -0,0 +1,124 @@
+---
+title: April '22 Community Gems
+date: 2022-04-26
+description: >
+  A roundup of technical Q&A's from the DVC and CML community. This month: CML
+  updates, working with multiple datasets, using DVC stages, and more.
+descriptionLong: >
+  A roundup of technical Q&A's from the DVC and CML community. This month: CML
+  updates, working with multiple datasets, using DVC stages, and more.
```

Comment:
long/short seems same? (feel free to resolve in intentional).

Also prefer *sans* `'`

```suggestion
  A roundup of technical Q&As from the DVC and CML community. This month: CML
  updates, working with multiple datasets, using DVC stages, and more.
descriptionLong: >
  A roundup of technical Q&As from the DVC and CML community. This month: CML
  updates, working with multiple datasets, using DVC stages, and more.
```